### PR TITLE
Improve reset handler

### PR DESCRIPTION
**Pull request by FeralChild64**
# Description

Few minor reset handler improvements
- displayed text no longer states "quitting", as we are restarting the emulator, not quitting it
- displayed text is now translatable
- blinking text cursor is now hidden
- changed video mode to 80x25 - to give more space to translators, and because I think it looks nicer

# Manual testing

Tested using 4DOS 8.00 - launch `4DOS.COM`, execute `REBOOT` command. For some reason this does not work with PCjr machine type (emulated machine hangs; it, however, works on Tandy, or even Hercules), but this is not a regression - the reboot handler is not even called on PCjr configuration, and the behavior on 0.81.0 is the same.

# Checklist

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [x] commented on the particularly hard-to-understand areas of my code.
- [ ] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [x] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

File: src/ints/bios.cpp [hunk 1]
```diff
@@ -1010,22 +1012,53 @@ static Bitu Default_IRQ_Handler()
 	return CBRET_NONE;
 }
 
-static Bitu Reboot_Handler(void) {
-	// switch to text mode, notify user (let's hope INT10 still works)
-	reg_ax = 0;
+static Bitu Reboot_Handler()
```
Comment (johnnovak) [inline src/ints/bios.cpp:1015]: `reboot_handler` while you're at it.

**Review by johnnovak — APPROVED** [review]
Looks good at the code level @FeralChild64. Maybe @weirddan455 can also take a look.

File: src/ints/bios.cpp [hunk 2]
```diff
@@ -1446,6 +1482,11 @@ class BIOS final : public Module_base{
 	}
 };
 
+void BIOS::AddMessages()
+{
+	MSG_Add("BIOS_REBOOT_REQUESTED", "Reboot requested...");
```
Comment (johnnovak) [inline src/ints/bios.cpp:1487]: Printing out the following would be cool, but not worth the complication if it's too difficult.

```
Rebooting in 3 seconds...
```

Then decrement 3 to 2, then to 1, and then reboot (with 1 second pauses).

**Review by weirddan455 — APPROVED** [review]
Looks like a good improvement to me.

File: src/ints/bios.cpp (hunk 1 above)
Comment (FeralChild64) [reply src/ints/bios.cpp]: OK, changed.

File: src/ints/bios.cpp (hunk 2 above)
Comment (FeralChild64) [reply src/ints/bios.cpp]: Not difficult at all. Changed.

**Pull request merged**